modules/nf-core/gatk4/cnnscorevariants/main.nf

```diff
@@ -3,7 +3,7 @@ process GATK4_CNNSCOREVARIANTS {
     label 'process_low'
 
     //Conda is not supported at the moment: https://github.com/broadinstitute/gatk/issues/7811
-    container "nf-core/gatk:4.4.0.0" //Biocontainers is missing a package
+    container "nf-core/gatk:4.5.0.0" //Biocontainers is missing a package
```

Comment:
Was the new container pushed to quay.io?

Reply:
No, that's what I was asking over on slack in #modules

Reply:
oh sorry, I checked my emails before Slack